docs/standard/generics/index.md

```diff
@@ -109,7 +109,7 @@ ms.author: "mairaw"
   
 -   Generic delegates enable type-safe callbacks without the need to create multiple delegate classes. For example, the <xref:System.Predicate%601> generic delegate allows you to create a method that implements your own search criteria for a particular type and to use your method with methods of the <xref:System.Array> type such as <xref:System.Array.Find%2A>, <xref:System.Array.FindLast%2A>, and <xref:System.Array.FindAll%2A>.  
   
--   Generics streamline dynamically generated code. When you use generics with dynamically generated code you do not need to generate the type. This increases the number of scenarios in which you can use lightweight dynamic methods instead of generating entire assemblies. For more information, see How to: Define and Execute Dynamic Methods and DynamicMethod.  
+-   Generics streamline dynamically generated code. When you use generics with dynamically generated code you do not need to generate the type. This increases the number of scenarios in which you can use lightweight dynamic methods instead of generating entire assemblies. For more information, see [How to: Define and Execute Dynamic Methods and DynamicMethod](../../../docs/framework/reflection-and-codedom/how-to-define-and-execute-dynamic-methods.md).  
```

Comment:
@imba-tjd good catch! 
I was puzzled why the link text doesn't match the article title. Most probably, there are two links here:
- one you've added to **How to: Define and Execute Dynamic Methods**
- another one should be the xref link (`<xref:System.Reflection.Emit.DynamicMethod>`) to https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.dynamicmethod

What do you think?

Reply:
Yeah... I think you are right. Thanks, I will try to modify as you said.

Reply:
Cool! Thanks!